### DISCUSSION TITLE
[BUG]Error "Connect-PnPOnline: Unknown Status: Unexpected" when using OSLogin switch

### DISCUSSION THREAD

**Discussion by ztrhgf** [pnp/powershell#4505] [General]
### Reporting an Issue or Missing Feature
I need to use **OSLogin** to authenticate via FIDO2, but I am unable to do so because it immediately throws an error. App created via Register-PnPAzureADApp is used to authenticate.

**EDIT: on different computer this is working fine, so maybe some cache or?**

### Expected behavior 
I expect some kind of window where I can authenticate

### Actual behavior 
![image](https://github.com/user-attachments/assets/23b72eb3-9231-4317-a12d-48cc2c8be590)

### Steps to reproduce behavior
Connect-PnPOnline -Url <redacted> -OSLogin -ClientId <AppCreatedVia:Register-PnPAzureADApp>

### What is the version of the Cmdlet module you are running?
2.99.28, 2.12.14

### Which operating system/environment are you running PnP PowerShell on?
Windows 10

**Comment by gautamdsheth** [comment]
Hmm, if it works on a different machine, would suggest you to check the differences between those 2 machines, maybe there's some issue with the user account or some policy .
